example_test.go

```diff
@@ -1,3 +1,5 @@
+// +build ignore
```

Comment:
🤔

Reply:
leave out I think, keep the examples compiled and ignore the leakage.